x/gov/keeper.go

```diff
@@ -48,6 +50,7 @@ func ParamKeyTable() params.KeyTable {
 }
 
 // Governance Keeper
+// Not safe to be held by other keepers, exposes Router().
```

Comment:
Do we need to? Can it be private?

Reply:
Alternatively there could be a good case for an `internal` package here